wire-library/docs/wire_compiler.md

```diff
@@ -236,6 +236,38 @@ wire {
 }
 ```
 
+### Version Matching
+
+Members may be declared with `wire.since` and `wire.until` options. For example, these options
```

Comment:
nit: might be great to include a sentence or two with the problem statement, i.e. what are these extensions trying to solve.

Reply:
fixed